packages/gce-disk-expand/packaging/debian/changelog

```diff
@@ -1,3 +1,9 @@
+gce-disk-expand (1:20200428.00-g1) stable; urgency=medium
```

Comment:
we don't edit these files anymore - they will be overwritten during build anyway